README.md

```diff
@@ -267,18 +267,23 @@ func main() {
 		// handle error
 	}
 
-	header := http.Header{
+	header := ws.HandshakeHeaderHTTP(http.Header{
```

Comment:
@gobwas This PR coming along good 👍 . Could you also update other examples for sending HTTP headers? since `ws.UpgradeHTTP(r, w)` doesn't have three parameter headers.

Reply:
Yeah, I fell that this changes almost done. I will check examples after all =)
Thank you!